sass/components/_icon-button-common.scss

```diff
@@ -0,0 +1,120 @@
+.IconButton {
+    display: inline-flex;
+    justify-content: center;
+    align-items: center;
+    position: relative;
+    padding: 10px;
+    width: 40px;
+    height: 40px;
+    font-size: 14/10*1rem;
+    line-height: 1;
+    color: color(--center-channel-color-rgb, 0.56);
+    border-radius: 4px;
+    border: none;
+    outline: none;
+    background: color(--center-channel-color-rgb, 0);
+    -webkit-tap-highlight-color: rgba(0,0,0,0);
+    overflow: hidden;
+    cursor: pointer;
+
+    // elements
+    .IconButton_icon {
+        color: color(--center-channel-color-rgb, 0.56);
+    }
+
+    // states
+    // - ::before used for inside border
+    &::before{
+        content: "";
+        position: absolute;
+        top: 0;
+        left: 0;
+        bottom: 0;
+        right: 0;
+        border-radius: 4px;
+        border: solid 2px transparent;
+        z-index: 0;
+    }
+
+    &:hover,
+    &.hover {
+        background: color(--center-channel-color-rgb, 0.08);
```

Comment:
Is this `color` function something we built or is it built in? I was supposed to create a ticket to make a helper like this :p

Reply:
Whoops. It seems like it does not exist. Probably copied from figma. Will fix.